projects/jamaica/README.md

```diff
@@ -0,0 +1,36 @@
+
+# Jamaica
+
+This project is configured to download/prepare/build a complete Pelias installation for the country of Jamaica.
+
+It's designed to be a larger example of using Pelias.
```

Comment:
Can we please remove this line of text?
We already have projects for Australia and the whole Planet, so this might confuse first-time users.